models/staking/ethereum/staking_ethereum_entities.sql

```diff
@@ -14,8 +14,8 @@ WITH contracts AS (
     SELECT lower(trim(address)) as address, trim(entity) as entity, trim(category) as category
     FROM
     (VALUES
-    ('0xdcd51fc5cd918e0461b9b7fb75967fdfd10dae2f', 'RocketPool (Minipool)', 'Staking Pools')
-    , ('0x1cc9cf5586522c6f483e84a19c3c2b0b6d027bf0', 'RocketPool (Minipool)', 'Staking Pools')
+    ('0xdcd51fc5cd918e0461b9b7fb75967fdfd10dae2f', 'RocketPool', 'Liquid Staking')
+    , ('0x1cc9cf5586522c6f483e84a19c3c2b0b6d027bf0', 'RocketPool', 'Liquid Staking')
```

Comment:
```suggestion
    ('0xdcd51fc5cd918e0461b9b7fb75967fdfd10dae2f', 'Rocket Pool', 'Liquid Staking')
    , ('0x1cc9cf5586522c6f483e84a19c3c2b0b6d027bf0', 'Rocket Pool', 'Liquid Staking')
    ,('0x2fb42ffe2d7df8381853e96304300c6a5e846905', 'Rocket Pool', 'Liquid Staking')
```
@hildobby The [latest Rocket Pool upgrade](https://discord.com/channels/405159462932971535/405163979141545995/1096244455150989362) created a new contract address, so adding it here since someone pointed me to this open PR. Happy to draft my own if preferred. I also changed the nomenclature to the preferred `Rocket Pool` over `RocketPool`. Cheers!

Reply:
Just wanted to confirm that 2 separate words is RP's preference.

As an aside -- _love_ your ETH2-staking dashboard.

Reply:
ty to both of you @acontry @Valdorff !! I wanted to add the new address but had not looked for it yet! And ofc, the preferred nomenclature is the one I want to use. Going to wrap up this PR so it can get merged asap :)